src/frontend/components/PlivoDialer.js

```diff
@@ -0,0 +1,106 @@
+import React from 'react';
+import {FloatingActionButton, FontIcon} from 'material-ui';
+
+export default class PlivoDialer extends React.Component {
+  static propTypes = {
+    endpointUsername: React.PropTypes.string,
+    endpointPassword: React.PropTypes.string,
+    number: React.PropTypes.string
+  }
+
+  state = {
+    displayed: true,
+    plivoCallInProgress: false,
+    plivoStatusText: null
+  }
+
+  registerCallbacks() {
+    Plivo.onWebrtcNotSupported = () => {
```

Comment:
I think we should make it so that instead of making the telephone number itself a tel: link, let's show always show this call icon and if webrtc is supported, use plivo, and if not, make the call icon a tel: link.  I can't think of a reason anyone would ever use a tel: link if webrtc is available and I think most people won't think to click the phone number/they will accidentally click it and get confused.